examples/scrolly-tile/scrolly-tile.cpp

```diff
@@ -107,6 +112,10 @@ using tile_callback = uint8_t (*)(uint8_t tile, uint8_t x, uint8_t y, void *args
 uint32_t prng_lfsr = 0;
 const uint16_t prng_tap = 0x74b8;
 
+uint32_t max(uint32_t a, uint32_t b) {
+  return a > b ? a : b;
+}
+

```

Comment:
Think you want `std::max` from `#include <algorithm>`

Reply:
Didn't know that existed! 😄

Reply:
You can probably drop this, plus the "highscore" variable altogether and just do:

```c++
if (player_progress > saveData.highscore) {
    saveData.highscore = player_progress;
}
```

No need to keep another copy of the score, the struct has you covered.

Reply:
Hmm, yeah... should've read the rest! Anyway, that should really be `save_data` for consistency.

Reply:
Need to update it when displaying the highscore as well then

Reply:
I'll fix this tomorrow I think